### ISSUE TITLE
The keychain state is stored with the wrong accessibility

### ISSUE THREAD

**Issue by eoger** [mozilla/application-services#3083]
**TL;DR**: Send Tab on Firefox iOS v25.0 does not work if your device is locked (and hasn't received tabs succesfully recently).

The handler for Send Tab push messages tries to get the FxA state from the keychain:

https://github.com/mozilla/application-services/blob/4d6778fca195591e95b14c1b3c8d5edb056e0b39/components/fxa-client/ios/FxAClient/FxAccountStorage.swift#L17

However we're not specifying an accessibility! By default on iOS it is `kSecAttrAccessibleWhenUnlocked`, which means we can't read the FxA state till the device is unlocked and explains the problem I describe in the beginning of this ticket.

When writing the patch, we also need to ensure current keychain items are migrated to the proper accessibility.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/SYNC-1236)
┆Epic: a-s consumer issues
┆Sprint: SYNC - end 2020-05-08

**Comment by rfk** [comment]
Excellent find 🚀
